td.vue/src/service/ajv-SchemaVerification.js

```diff
@@ -0,0 +1,18 @@
+import Ajv from 'ajv';
```

Comment:
Here are some changes to get this working for version 1.x threat models
It still needs more work for version 2.x threat models, will do that later this week:

```
% git diff td.vue/src/service/ajv-SchemaVerification.js
--- a/td.vue/src/service/ajv-SchemaVerification.js
+++ b/td.vue/src/service/ajv-SchemaVerification.js
@@ -1,18 +1,18 @@
 import Ajv from 'ajv';
-import { schema } from '../assets/threat-model-schema';
+import { schema } from '@/assets/threat-model-schema';
 
-const ajv = new Ajv({strict : false});
+const ajv = new Ajv();
 const validate = ajv.compile(schema);
 
 export const isValidSchema = (jsonFile) => {
 
-
     const valid = validate(jsonFile);
 
-    if (valid.errors) {
-        console.error("Failed to validate", validate.errors);
+    if (!valid) {
+        console.error('Failed to validate', validate.errors);
         return false;
     } else {
+        console.debug('Schema validate success');
         return true;
     }
 }
```